gulpfile.babel.js

```diff
@@ -1,91 +0,0 @@
-/**
- * Gulp config.
- *
- * Site Kit by Google, Copyright 2019 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/**
- * External dependencies
- */
-import gulp from 'gulp';
-import requireDir from 'require-dir';
-import runSequence from 'run-sequence';
-import livereload from 'gulp-livereload';
-import del from 'del';
-
-requireDir( './gulp-tasks' );
-
-/**
- * Gulp task to run all SVG processes in a sequential order.
- */
-gulp.task( 'build', () => {
-	runSequence(
-		'webpack',
-		'svg',
-		'imagemin'
-	);
-} );
-
-/**
- * Gulp task to watch for file changes and run the associated processes.
- */
-gulp.task( 'watch', () => {
-	livereload.listen( { basePath: 'dist' } );
-	gulp.watch( './assets/sass/**/*.scss', [ 'build' ] );
-	gulp.watch( './assets/svg/**/*.svg', [ 'build' ] );
-	gulp.watch( './assets/js/*.js', [ 'build' ] );
-	gulp.watch( './assets/js/modules/**/*.js', [ 'build' ] );
-} );
```

Comment:
A few tasks like these ones are missing from the new gulp file, can you clarify why you removed these? E.g. I can't find `build` and `watch`? I saw in your PR description you removed some, but would be great if you give a bit more context on why. Did you remove them because they're not used by any NPM scripts?

Reply:
@felixarntz those tasks have been removed because we don't use it anymore. Gulp tasks like `webpack`, `build`, `watch`, `local` and `browser-sync` aren't needed anymore. It has been replaced with webpack itself and webpack-cli commands.